rectools/models/lightfm.py

```diff
@@ -131,6 +149,13 @@ def __init__(
         self._model = model
         self.n_epochs = epochs
         self.n_threads = num_threads
+        self._recommend_n_threads = recommend_n_threads
+        self.recommend_n_threads = 0
+        if recommend_n_threads is not None:
+            self.recommend_n_threads = recommend_n_threads
+        elif num_threads > 0:
```

Comment:
By the docstring it's always >0, so we don't need this condition here

Reply:
fixed